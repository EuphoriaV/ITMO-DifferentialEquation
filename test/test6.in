4.14
(7.4 - (7.99 * x))
1.15
5.06