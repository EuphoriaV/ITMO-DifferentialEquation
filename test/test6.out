0.0356129